### ISSUE TITLE
Tests are failing

### ISSUE THREAD

**Issue by RunsFor** [tarantool/kafka#26]
Several tests are failing when run suggested `make test-run-with-docker` command. Common error is:

```
KafkaConnectionError: No connection to node with id 1001
```

Test results:
```
test_consumer.py::test_consumer_should_consume_msgs FAILED                                                                                             [  9%]
test_consumer.py::test_consumer_should_consume_msgs_from_multiple_topics FAILED                                                                        [ 18%]
test_consumer.py::test_consumer_should_completely_unsubscribe_from_topics FAILED                                                                       [ 27%]
test_consumer.py::test_consumer_should_partially_unsubscribe_from_topics FAILED                                                                        [ 36%]
test_consumer.py::test_consumer_should_log_errors PASSED                                                                                               [ 45%]
test_consumer.py::test_consumer_should_log_debug PASSED                                                                                                [ 54%]
test_consumer.py::test_consumer_should_log_rebalances Unclosed AIOKafkaProducer
producer: <aiokafka.producer.producer.AIOKafkaProducer object at 0x10bca8d90>
Unclosed AIOKafkaProducer
producer: <aiokafka.producer.producer.AIOKafkaProducer object at 0x10bf2e5d0>
Unclosed AIOKafkaProducer
producer: <aiokafka.producer.producer.AIOKafkaProducer object at 0x10bf5b6d0>
Unclosed AIOKafkaProducer
producer: <aiokafka.producer.producer.AIOKafkaProducer object at 0x10bd63090>
FAILED                                                                                           [ 63%]
test_consumer.py::test_consumer_should_continue_consuming_from_last_committed_offset FAILED                                                            [ 72%]
test_producer.py::test_producer_should_produce_msgs FAILED                                                                                             [ 81%]
test_producer.py::test_producer_should_log_errors PASSED                                                                                               [ 90%]
test_producer.py::test_producer_should_log_debug PASSED                                                                                                [100%]
```

**Comment by RunsFor** [comment]
Environment:

macOS 10.15.5 (19F101)
Python 3.7.8
Docker 19.03.8

**Comment by RepentantGopher** [comment]
Should be fixed in https://github.com/tarantool/kafka/pull/30

**Comment by RepentantGopher** [comment]
Fixes was released in 1.3.0 version https://github.com/tarantool/kafka/releases/tag/1.3.0
